insights/terraform/projects/bb2/services/common/modules/lambda/main.tf

```diff
@@ -0,0 +1,52 @@
+data "aws_caller_identity" "current" {}
+
+data "aws_kms_key" "bucket_cmk" {
+  key_id = "alias/bfd-insights-bb2-cmk"
+}
+
+# Zip File containing Lambda script
+data "archive_file" "zip-archive-format-firehose-logs" {
+  type        = "zip"
+  source_file = "${path.module}/lambda_src/lambda_function.py"
+  output_path = "${path.module}/lambda_function.zip"
```

Comment:
Should we add a `.gitignore` for zips here?

Reply:
TY for catching this. Forgot to add the file before.